etc/containerpilot.json

```diff
@@ -24,12 +24,13 @@
     {
       "name": "snapshot_check",
       "command": "python /usr/local/bin/manage.py snapshot_task",
-      "frequency": "10s",
```

Comment:
Also: at some point I'd like to look into improving this so we do incremental backups rather than full snapshots.